scripts/lib/zulip_tools.py

```diff
@@ -37,6 +37,14 @@
 MAGENTA = '\x1b[35m'
 CYAN = '\x1b[36m'
 
+def overwrite_symlink(src, dst):
+    # type: (Union[bytes, str], Union[bytes, str]) -> None
+    try:
+        os.symlink(src, dst)
+    except FileExistsError:
+        os.remove(dst)
+        os.symlink(src, dst)
```

Comment:
Can we make this atomic?  I don't think it's an improvement to lose atomicity to save a `subprocess` call; correctness is more important than a tiny bit of performance.

Reply:
Probably. Fun fact, though: `ln -nsf` *isn’t* atomic until coreutils 8.27, which is in Ubuntu 18.04 but not 17.10 and older!